tests/providerTest.js

```diff
@@ -185,6 +185,16 @@ describe('provider testing', () => {
         return { description: 'user created' }
       }
     },
+    'user doesn\'t exist': (setup, parameters) => {
+      if (setup) {
+        const result = deleteUser(parameters.username)
+
+        chai.assert.strictEqual(
+          result.status, 200, `deleting user '${parameters.username}' failed`
+        )
```

Comment:
```
Message:
      deleting user 'test123' failed: expected 204 to equal 200
```
User deletion response code can either be `200` or `204`

Reply:
I think `.include` could help
```
chai.assert.include(
          [200, 204],result.status, `deleting user '${parameters.username}' failed`
        )
```